### PR TITLE
Applied minimun liquidity

### PR DESCRIPTION
**Pull request by psy2848048**
## Summary of changes

- Applied minimun liquidity
- Transfer the # of "minimum liquidty" to the pair contract

## Report of required housekeeping

- [ ] Github issue OR spec proposal link
- [x] Wrote tests
- [ ] Added a relevant changelog entry to CHANGELOG.md

----

## (FOR ADMIN) Before merging

- [ ] Added appropriate labels to PR
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" (coding standards)
- [ ] Confirm added tests are consistent with the intended behavior of changes
- [ ] Ensure all tests pass

### CODE REVIEW

**Comment by JoowonYun** [comment]
What do you think about checking the minimum liquidity when withdraw to support pairs that currently have liquidity?

File: contracts/dezswap_pair/src/contract.rs [hunk 1]
```diff
@@ -276,12 +278,34 @@ pub fn provide_liquidity(
...
+        .try_into()
         {
             Ok(share) => share,
             Err(e) => return Err(ContractError::ConversionOverflowError(e)),
-        }
+        };
+
+        // MINIMUM_LIQUIDITY_AMOUNT is for the contract's stake
+        messages.push(CosmosMsg::Wasm(WasmMsg::Execute {
+            contract_addr: deps
```
Comment (nowooj) [inline contracts/dezswap_pair/src/contract.rs:291]: pair_info.liquidity_token.to_string()

Comment (psy2848048) [reply contracts/dezswap_pair/src/contract.rs:291]: Isn't it for use `liquidity0000` mock address in testing

**Review by nowooj — APPROVED** [review]

**Review by jhlee-young — APPROVED** [review]
LGTM

**Pull request merged**